libs/wingsdk/src/std/json.ts

```diff
@@ -32,6 +32,32 @@ export class Json {
     throw new Error("Macro");
   }
 
+  /**
+   * Returns the values from the Json.
+   *
+   * @macro (Object.values($args$))
+   *
+   * @param json to get values from
+   * @returns the values from the Json as array of Json
+   */
+  public static values(json: Json): Json[] {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Returns the entries from the Json.
+   *
+   * @macro (Object.entries($args$))
+   *
+   * @param json to get values from
```

Comment:
```suggestion
   * @param json to get entries from
```

Also, I guess we should mention here that it should be a map json and not just any arbitrary one.

Reply:
Then it should be the same also for `keys()` and `values()`. Should I change all three?

Reply:
I updated the same also for `keys()` and `values()`, can you take a look and let me know what you think?

Reply:
I updated the same also for `keys()` and `values()`. Can you take a look and let me know what you think?